README.md

```diff
@@ -4,13 +4,17 @@
 
 This Cordova Plugin allows you to create maximum 4 Android Static Shortcuts saving in memory the selected one and get this action to do something in the base application. Tested on ionic 1, 2 and 3 based projects.
 
-## Install and configuration
+## Installing plugin
+
+Install the plugin with the following command:
 
-First, install the plugin with the following command:
 ````
 cordova plugin add https://github.com/kunder-lab/cordova-plugin-kunder-android-shortcuts.git
 ````
-Then, create android-shortcuts.json file with all the information about the shortcuts you want to create (maximum 4 shortcuts):
+
+## Static shortcuts configuration
+
+Create android-shortcuts.json file in project's root folder with all the information about the shortcuts you want to create (maximum 4 shortcuts):
```

Comment:
[S:0] Se sugiere dejar el comienzo de la frase como: "Create a JSON file called `android-shortcuts.json` in the root folder of the project..." o "Create the `android-shortcuts.json` file in the root folder of the project... ".